app/api/v2/admin.py

```diff
@@ -81,13 +82,13 @@ def on_post(self, req, res, **kwargs):
             filter(Listing.token_address == contract_address). \
             first()
         if _listing is not None:
-            raise InvalidParameterError("contract_address already exist")
+            raise DataConflictError("contract_address already exist")
 
         _executable_contract = session.query(ExecutableContract). \
             filter(ExecutableContract.contract_address == contract_address). \
             first()
         if _executable_contract is not None:
-            raise InvalidParameterError("contract_address already exist")
+            raise DataConflictError("contract_address already exist")
```

Comment:
Added error response type so that user can determine if posted token has already registered or not.